graphics/macgui/macmenu.cpp

```diff
@@ -280,6 +291,9 @@ MacMenu *MacMenu::createMenuFromPEexe(Common::PEResources *exe, MacWindowManager
 	if (gDebugLevel > 5)
 		menu->printMenu();
 
+	if (wm->_mode & Graphics::kWMModeWin95)
+		menu->enableAllMenus();
```

Comment:
What is the reasoning behind this change?

Reply:
I was asked to do this because apparently that's the required behavior. Some menus which should have been disabled according to the data in the PE EXE were enabled in the actual game. 

Relevant discussion on discord: https://discord.com/channels/581224060529148060/843917001307193354/994610356972421240